tools/CMakeLists.txt

```
@@ -0,0 +1,6 @@
+ADD_EXECUTABLE( PodioEventOverview PodioEventOverview.cpp )
+TARGET_LINK_LIBRARIES( PodioEventOverview podioRootIO podioSioIO podio )
+
+
```

```suggestion
INSTALL(TARGETS PodioEventOverview DESTINATION ${CMAKE_INSTALL_BINDIR})
```
We also need to install the binary to `bin` to make it usable after it has been built.